frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -835,7 +835,7 @@ function ReaderRolling:onGotoXPointer(xp, marker_xp)
         -- where xpointer target is (and remove if after 1s)
         local screen_y, screen_x = self.ui.document:getScreenPositionFromXPointer(marker_xp)
         local doc_margins = self.ui.document:getPageMargins()
-        local marker_h = Screen:scaleBySize(self.ui.font.font_size * 1.1 * self.ui.font.line_space_percent * (1/100))
+        local marker_h = Screen:scaleBySize(self.ui.font.font_size * 1.1 * self.configurable.line_spacing * (1/100))
```

Comment:
(So, we won't know from reading the code which module handles line spacing - but we'll know it's a "configurable" from the bottom menu. So, I guess I'm ok with that.)